### DISCUSSION TITLE
Delivery Note and "Orders" pages

### DISCUSSION THREAD

**Discussion by SrGatoo** [crater-invoice-inc/crater#1048] [Ideas]
I've tried the docker and, althought installation wasn't flawless...

Seems like a pretty good software!

However, there's a few catchs for me, so I'll have to go back to my usual software.

We've got estimates, billings... it'll be REALLY nice to have a page where you can track the "active orders" and another one for the "delivery notes"

For example, an estimate gets accepted, we can convert it to an order and prepare it. Once is done, we can convert it to a delivery note that can be sended to our client and once sent.. converted into the invoice.

Also, please, add the option to ALWAYS use TAX by default!

Thanks for your time

**Comment by tenent007963** [comment]
Feature request has been raised at #1079 , hopefully someone will notice it and implement it
